transformations/gender_neopronouns/transformation.py

```diff
@@ -0,0 +1,116 @@
+from typing import List
+
+import spacy
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+from .constants import TAGS
+
+
+class GenderNeoPronouns(SentenceOperation):
+    """This transformation performs **_grammatically correct substitution_**
+    of the gendered pronouns he/she in a given sentence with their
+    neopronoun counterparts.
+    """
+
+    tasks = [TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
+
```

Comment:
Please provide relevant keywords, using [this list](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/docs/keywords.md)